commands

```diff
@@ -30,9 +30,16 @@ case "$1" in
     mkdir -p "$SERVICE_ROOT/config" || dokku_log_fail "Unable to create service config directory"
     echo -e "[mysqld]\nperformance_schema = 0" > "$SERVICE_ROOT/config/disable_performance_schema.cnf"
     rootpassword=$(openssl rand -hex 8)
-    password=$(openssl rand -hex 8)
     echo "$rootpassword" > "$SERVICE_ROOT/ROOTPASSWORD"
```

Comment:
I don't think so. But setting root's password via a similar mechanism might not be a bad thing. That said, I'm not sure you could set it to "" (blank) and things would work.